# --- ROCK-COPYRIGHT-NOTE-BEGIN ---
# 
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# Please add additional copyright information _after_ the line containing
# the ROCK-COPYRIGHT-NOTE-END tag. Otherwise it might get removed by
# the ./scripts/Create-CopyPatch script. Do not edit this copyright text!
# 
# ROCK Linux: rock-src/package/fake/xmame/makefile-settings.patch
# ROCK Linux is Copyright (C) 1998 - 2005 Clifford Wolf
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# 
# --- ROCK-COPYRIGHT-NOTE-END ---

--- ./makefile.unix.orig	2004-11-24 20:47:30.000000000 +0100
+++ ./makefile.unix	2004-11-24 22:11:19.031067264 +0100
@@ -89,7 +89,7 @@
 # just as well.  However, stay away from the `cc' Ultrix compiler if 
 # possible.
 
-CC	= gcc
+# CC	= gcc
 # CC    = cc
 # CC	= icc
 # CC	= c89
@@ -101,7 +101,7 @@
 # Compiler for host compilations in cross-compiling environments (used 
 # in src/unix/unix.mak for m68k).
 # HOST_CC=$(CC)
-HOST_CC = gcc
+# HOST_CC = gcc
 
 
 ###########################################################################
@@ -110,7 +110,7 @@
 
 # If you want to use whatever CFLAGS are currently set in your 
 # environment, then comment this out.
-CFLAGS =
+# CFLAGS =
 
 
 ###########################################################################
@@ -119,7 +119,7 @@
 ###########################################################################
 
 # GCC on x86
-CFLAGS = -O -Wall -Wno-unused
+# CFLAGS = -O -Wall -Wno-unused
 
 # GCC on x86 with some optimizations
 # CFLAGS = -O2 -Wall -Wno-unused -mcpu=i686 -fomit-frame-pointer \
@@ -429,7 +429,7 @@
 
 # Does your platform have ranlib?  Leave this untouched unless you are
 # using IRIX, Unixware 7, or the Intel C++ Compiler.
-RANLIB = ranlib
+# RANLIB = ranlib
 # RANLIB = true
 
 
@@ -504,7 +504,7 @@
 # Select destination directory for your compiled program, manual page,
 # and binary distribution.
 ifndef PREFIX
-PREFIX = /usr/local
+PREFIX = /usr
 endif
 BINDIR = $(PREFIX)/bin
 MANDIR = $(PREFIX)/man/man6
@@ -520,7 +520,9 @@
 ###########################################################################
 
 # i386, GNU asm
+ifndef MY_CPU
 MY_CPU = i386
+endif
 
 # i386, no asm -- needed for the Intel C++ compiler, which does not fully
 # understand GCC's inline assembly syntax, though you may still enable 
@@ -611,7 +613,7 @@
 
 # Enable the standard joystick driver, which should work in Linux, OpenBSD, 
 # NetBSD, and FreeBSD.
-# JOY_STANDARD = 1
+JOY_STANDARD = 1
 
 # Linux FM-TOWNS game pad joystick emulation support.  Thanks to Osamu 
 # Kurati.
@@ -624,7 +626,7 @@
 # JOY_PS2 = 1
 
 # SDL joystick support.
-# JOY_SDL = 1
+JOY_SDL = 1
 
 # This will enable the use of the Happ Controls UGCI(tm) USB devices'
 # Coin/Start inputs using the libugci library available from
@@ -650,10 +652,10 @@
 # Choose any additonal sound drivers you want to include, besides the 
 # native sound driver for your system.
 # SOUND_ESOUND = 1
-# SOUND_ALSA = 1
+SOUND_ALSA = 1
 # SOUND_ARTS_TEIRA = 1
 # SOUND_ARTS_SMOTEK = 1
-# SOUND_SDL = 1
+SOUND_SDL = 1
 # SOUND_WAVEOUT = 1
 
 
@@ -665,10 +667,10 @@
 # use SDL.
 
 # X11 (including Glide & OpenGL if enabled below)
-DISPLAY_METHOD = x11
+# DISPLAY_METHOD = x11
 
 # SDL library 
-# DISPLAY_METHOD = SDL
+DISPLAY_METHOD = SDL
 
 # SVGALib, only supported under Linux
 # DISPLAY_METHOD = svgalib
